src/styles/getMuiTheme.js

```diff
@@ -224,4 +245,11 @@ export default function getMuiTheme(baseTheme, muiTheme) {
       borderColor: palette.borderColor,
     },
   }, muiTheme);
+
+  const transformers = computedTheme.transformers ? computedTheme.transformers :
```

Comment:
it works either way. I'm alright with it. With something like C# I would be paranoid too since IL has `hide by signature`, in other words signature is documentation. but with a language this dynamic. I don't see any difference actually i just think this is easier:

```js
getMuiTheme(null, null, [stuff, stuff2]);
```

than

```js
getMuiTheme(null, {transformers: [stuff, stuff2]});
```